scripts/importer/importer.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+"""
+Swefreq data importer.
+
+Requires apt-packages:
+    - python3-pip
+    - sudo apt-get install libmysqlclient-dev
+
+as well as pip3 packages:
+    - mysqlclient
+    - peewee-2.10.2
+"""
+
+from data_importer.dbsnp_importer import DbSNPImporter
+from data_importer.reference_set_importer import ReferenceSetImporter
+from data_importer.old_db_importer import OldDbImporter
+from data_importer.raw_data_importer import RawDataImporter
+
+if __name__ == '__main__':
+
+    import os
+    import argparse
+    import logging
+
+    parser = argparse.ArgumentParser( description = __doc__ ,
+             formatter_class = argparse.ArgumentDefaultsHelpFormatter )
+
+    parser.add_argument("--batch_size", type=int, default=5000,
+                        help = ("Where batch insertion is possible, use this number of"
+                                " inserts per batch."))
+    parser.add_argument("--limit_chrom", default=None,
+                        help = "Limit chromosome to insert into the database.")
+    parser.add_argument("--data_dir", default=os.path.join(os.path.dirname(__file__),
+                                                           "downloaded_files"),
+                        help = "Default directory to download and look for files.")
+
+    # Reference versions
+    parser.add_argument("--gencode_version", default=19, type=int,
+                        help = "Gencode version to download and use.")
+    parser.add_argument("--ensembl_version", default="homo_sapiens_core_75_37",
+                        help = "Ensembl database to connect to.")
+    parser.add_argument("--dbnsfp_version", default="2.9.3",
+                        help = "dbNSFP version to download and use.")
+    parser.add_argument("--dbsnp_version", default="b150",
+                        help = "dbSNP version to download and use.")
+    parser.add_argument("--dbsnp_reference", default="GRCh37p13",
+                        help = "Which reference the dbSNP should be aligned to.")
+
+    # Dataset connections and naming
+    parser.add_argument("--dataset", default="",
+                        help="Which dataset to connect imported data to.")
+    parser.add_argument("--version", default="latest",
```

Comment:
I don't think this should have a default. It shouldn't be too easy to overwrite the latest version with new data.

Reply:
latest is no longer default